src/commands/upload.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  dashboard-name:
+    type: string
+    default: dashboard
+    description: |
+      Name of the json(net) dashboard file. It is assumed that they share the same name.
+      Defaults to 'dashboard'.
+  dashboard-path:
+    type: string
+    default: monitoring
+    description: |
+      Specify the path to where json dashboard is available. Omit any trailing /.
+      Defaults to 'monitoring'.
+  folder-uid:
+    type: string
+    description: |
+      Folder uid from Grafana.
+      Uid can be taken from 'HOST/dashboards/f/{UID}'
+  overwrite-existing:
+    type: boolean
+    default: true
+    description: |
+      Whether to overwrite existing dashboard matching name/uid.
+      Defaults to 'true'.
+steps:
+  - run:
+      name: Update remote dashboard
+      working_directory: <<parameters.dashboard-path>>
+      command: |
+        BODY="{
+              \"dashboard\": $(jq . <<parameters.dashboard-name>>.json),
+              \"folderUid\": \"<<parameters.folder-uid>>\",
+              \"overwrite\": <<parameters.overwrite-existing>>
```

Comment:
Is there a way to add message as well, perhaps using the commit-log or even just referencing the commit SHA?

Reply:
yes, that is possible